docs/source/conf.py

```diff
@@ -0,0 +1,187 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+import sphinx_rtd_theme
+import skcosmo
+
+# -- Project information -----------------------------------------------------
+
+# The master toctree document.
+master_doc = 'index'
+
+project = "sklearn-COSMO"
+author = ", ".join(open("../../contributors.txt"))
```

Comment:
Did you forgot to `git add` this file? I don't see it

Reply:
Yes, yes I did. It is added now.